src/System.Private.CoreLib/shared/System/Globalization/JapaneseCalendar.Win32.cs

```diff
@@ -93,10 +93,10 @@ public partial class JapaneseCalendar : Calendar
             Array.Resize(ref registryEraRanges, iFoundEras);
 
             // Sort them
-            Array.Sort(registryEraRanges, CompareEraRanges);
+            Array.Sort(registryEraRanges!, CompareEraRanges); // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
```

Comment:
Was the resize what confused the compiler here?

Reply:
Yes.  We lack an annotation that says "this ref will never be null upon return", so the compiler assumes that any call sites to Array.Resize may be setting the array to null, even though that'll never happen (to the contrary, it'll never be null even if it went in as null).
cc: @jcouv